features/jira.py

```diff
@@ -0,0 +1,27 @@
+from datetime import date
```

Comment:
It's generally a good idea to avoid naming your own modules the same as a package, just to avoid namespace clashes.

e.g. here I would call your .py file something different, because you are importing a library called jira

Technically what you've done is fine, mind you - I can't see that the namespaces would clash currently.

Another option to help with this is to import libraries with an alias e.g.

`from jira import JIRA as JR` 

But I wouldn't do that in this case, personally.